lisa/nic.py

```diff
@@ -182,13 +210,15 @@ def get_nic_driver(self, nic_name: str, store_driver: bool = True) -> str:
         assert_that(driver_name).described_as(
             f"sysfs entry contained no filename for device driver: {found_link}"
         ).is_not_equal_to("")
-        if store_driver:
-            nic.bound_driver = driver_name
+        nic.module_name = driver_name
         return driver_name
 
     def get_nic(self, nic_name: str) -> NicInfo:
         return self.nics[nic_name]
 
+    def get_nic_names(self) -> List[str]:
```

Comment:
1. It's not used out of the nics class, so it should be a private method.
2. Does it need to include lower as well?

Reply:
it is used in other places besides nics. 
no need include lower nics.